src/settings/JobProfiles/JobProfilesForm/JobProfilesForm.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+  injectIntl,
+} from 'react-intl';
+import { noop } from 'lodash';
+import { Field } from 'react-final-form';
+
+import {
+  Layer,
+  TextField,
+  TextArea,
+  Select,
+} from '@folio/stripes/components';
+import stripesFinalForm from '@folio/stripes/final-form';
+import {
+  FullScreenForm,
+  Preloader,
+} from '@folio/stripes-data-transfer-components';
+
+import { required } from '../../../utils';
+
+import css from './JobProfilesForm.css';
+
+const validate = values => {
+  const errors = {};
+
+  errors.name = required(values.name);
+  errors.mappingProfileId = required(values.mappingProfileId);
+
+  return errors;
+};
+
+const JobProfilesForm = props => {
+  const {
+    onCancel,
+    intl,
+    handleSubmit,
+    pristine,
+    submitting,
+    hasLoaded,
+    mappingProfiles,
+  } = props;
+
+  return (
+    <FormattedMessage id="ui-data-export.mappingProfiles.newProfile">
+      {contentLabel => (
+        <Layer
+          isOpen
+          contentLabel={contentLabel}
+        >
+          <FullScreenForm
+            id="job-profiles-form"
+            paneTitle={<FormattedMessage id="ui-data-export.jobProfiles.newProfile" />}
+            onSubmit={handleSubmit}
+            onCancel={onCancel}
+            isSubmitButtonDisabled={pristine || submitting}
```

Comment:
Please move it above callbacks.

Reply:
Please, could you clarify why it should be placed above callbacks?

Reply:
Just per our convention. I know it is not the application-wide but at least the majority of our codebase complies with it.